repo2docker/app.py

```diff
@@ -415,6 +415,16 @@ def _user_name_default(self):
         """,
     )
 
+    base_image = Unicode(
+        "buildpack-deps:bionic",
```

Comment:
```suggestion
        "docker.io/library/buildpack-deps:bionic",
```
Might as well close https://github.com/jupyterhub/repo2docker/issues/1146 at the same time